Authentication error message does not link to documentation

**Describe the bug**
```
version: v23.10.0
revision: d86643a0f743e6dd8cc9bed0fdbbeb256cc6a8c7


Flank encountered a 403 error when running on project flank-gradle (from /Users/nosacky/workspace/fladle/flank-gradle-creds.json). Please verify this credential is authorized for the project and has the required permissions.
Consider authentication with a Service Account https://github.com/Flank/flank#authenticate-with-a-service-account
or with a Google account https://github.com/Flank/flank#authenticate-with-a-google-account

Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
POST https://toolresults.googleapis.com/toolresults/v1beta3/projects/flank-gradle:initializeSettings
{
  "code": 403,
  "errors": [
    {
      "domain": "global",
      "message": "The caller does not have permission",
      "reason": "forbidden"
    }
  ],
  "message": "The caller does not have permission",
  "status": "PERMISSION_DENIED"
}

Total run duration: 0m  2s
```

**To Reproduce**

1. Delete auth tokens,
2. Run a command with Flank 23.10.0
3. See error


**Expected behavior**
The error message links to the Flank readme but the readme is empty.


Instead of this link:
https://github.com/Flank/flank#authenticate-with-a-service-account
We should use this link:
https://flank.github.io/flank/#authenticate-with-a-service-account
Instead of this link:
https://github.com/Flank/flank#authenticate-with-a-google-account
We should use this link:
https://flank.github.io/flank/#authenticate-with-a-google-account

**Additional context**
> Add any other context about the problem here.

--- a/test_runner/src/main/kotlin/ftl/client/google/GcToolResults.kt
+++ b/test_runner/src/main/kotlin/ftl/client/google/GcToolResults.kt
@@ -222,8 +222,8 @@ object GcToolResults {
 
 private val permissionDeniedErrorMessage = { projectId: String, projectIdSource: String?, message: String? ->
     """Flank encountered a 403 error when running on project $projectId${projectIdSource?.let {" (from $it)"} ?: ""}. Please verify this credential is authorized for the project and has the required permissions.
-Consider authentication with a Service Account https://github.com/Flank/flank#authenticate-with-a-service-account
-or with a Google account https://github.com/Flank/flank#authenticate-with-a-google-account
+Consider authentication with a Service Account https://flank.github.io/flank/#authenticate-with-a-service-account
+or with a Google account https://flank.github.io/flank/#authenticate-with-a-google-account
 
 $message
     """.trimIndent()

--- a/test_runner/src/test/kotlin/ftl/gc/GcToolResultsTest.kt
+++ b/test_runner/src/test/kotlin/ftl/gc/GcToolResultsTest.kt
@@ -60,8 +60,8 @@ class GcToolResultsTest {
     fun `getDefaultBucket on 403 error should throw exception with specific message - no source`() {
         val expected = """
             Flank encountered a 403 error when running on project $projectId. Please verify this credential is authorized for the project and has the required permissions.
-            Consider authentication with a Service Account https://github.com/Flank/flank#authenticate-with-a-service-account
-            or with a Google account https://github.com/Flank/flank#authenticate-with-a-google-account
+            Consider authentication with a Service Account https://flank.github.io/flank/#authenticate-with-a-service-account
+            or with a Google account https://flank.github.io/flank/#authenticate-with-a-google-account
             
             Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
             {
@@ -105,8 +105,8 @@ class GcToolResultsTest {
     fun `getDefaultBucket on 403 error should throw exception with specific message - with source`() {
         val expected = """
             Flank encountered a 403 error when running on project $projectId (from /Any/path/to/json.json). Please verify this credential is authorized for the project and has the required permissions.
-            Consider authentication with a Service Account https://github.com/Flank/flank#authenticate-with-a-service-account
-            or with a Google account https://github.com/Flank/flank#authenticate-with-a-google-account
+            Consider authentication with a Service Account https://flank.github.io/flank/#authenticate-with-a-service-account
+            or with a Google account https://flank.github.io/flank/#authenticate-with-a-google-account
             
             Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
             {

